src/main/java/seedu/address/model/ExpenseExpert.java

```diff
@@ -25,6 +27,7 @@
      */
     {
         expenses = new UniqueExpenseList();
+        budget = new Budget("0");
```

Comment:
I dont think it will break the application. Will review in v1.3. Additionally, adding new expense does not create new Budget object, it modifies the current Budget object existing in ExpenseExpert.

Reply:
can resolve in 1.3